plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftHiveMetastore.java

```diff
@@ -197,25 +196,25 @@
     @Inject
     public ThriftHiveMetastore(
             MetastoreLocator metastoreLocator,
-            HiveConfig hiveConfig,
             MetastoreConfig metastoreConfig,
+            @TranslateHiveViews Optional<Boolean> translateHiveViews,
```

Comment:
Why `Optional`? it's always either true or false

```suggestion
            @TranslateHiveViews boolean translateHiveViews)
```